core/classes.py

```diff
@@ -1,6 +1,79 @@
+from logging import Logger
+
 import discord
 from discord.ext import commands
 
+
 class Cog_Extension(commands.Cog):
-       def __init__(self,bot):
-            self.bot = bot
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+
+class Hybirdcmd_Aliases():
+    def __init__(self, lang, *cmd_names: str):
+
+        self.cmd_names = list(cmd_names)
+        self.aliases = {}
+        self.description = {}
+        self.brief = {}
+
+        for cmd_name in cmd_names:
+
+            aliases = []
+            for alias in lang.get(cmd_name+'.aliases', []):
+                alias = alias.lower()
+                if alias != cmd_name.lower():
```

Comment:
we can change that in the loc repo instead

Reply:
how about no need to change